aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -108,13 +106,36 @@ http://localhost/Speaker/Evaluations#SpeakerEvaluations
 ```
 Hash tags are useful when doing client side applications. They can be used for easy marking and searching in JavaScript for example.
 
+- - -
+
+### asp-area
+
+```asp-area``` sets the area name that ASP.NET core will use to set the appropriate route with.  Below are examples of how the area attribute causes a remapping of routes.  That is, by setting ```asp-area``` to Blogs will invoke the Areas functionality such that the directory Areas/Blogs will prefix the associated controllers and views for this anchor tag.
```

Comment:
* "core" to "Core"
* "... will use to set the appropriate route with." ... prob goes against what they're doing with future tense and "with" at the end of the sentence always gives me a tooth ache [but @tdykstra will like it :grin: (another inside joke :beer:)]. How about "... that ASP.NET Core uses to create the appropriate route to an area."

> That is, by setting ```asp-area``` to Blogs will invoke the Areas functionality such that the directory Areas/Blogs will prefix the associated controllers and views for this anchor tag.

Setting `asp-area` to `Blogs` prefixes the directory `Areas/Blogs` to the routes of the associated controllers and views for this anchor tag.